src/app/services/cell.service.ts

```diff
@@ -15,10 +15,16 @@ export class CellService extends GlobalService {
     super();
   }
 
-  getCells(): Observable<any> {
+  getCells(isOrdered: boolean): Observable<any> {
     const headers = this.getHeaders();
+
+    let orderedTxt = '';
+    if (isOrdered) {
+      orderedTxt = '?ordering=name';
+    }
+
     return this.http.get<any>(
-      this.url_cells,
+      this.url_cells + orderedTxt,
```

Comment:
In Angular the good way to add some url params is to use `Params` 

Example:
https://github.com/Volontaria/Website-Volontaria/blob/af32a285a901fb6c3240473369a0110a008dd28b/src/app/services/event.service.ts#L25

Reply:
I didn't know that, I will do it this way from now on. ;) Thank you !